Dockerfile

```diff
@@ -70,7 +70,7 @@ RUN apt-add-repository ppa:brightbox/ruby-ng \
   && apt-get install -y ruby2.6 ruby2.6-dev \
   && gem update --system 3.2.14 \
   && gem install bundler -v 1.17.3 --no-document \
-  && gem install bundler -v 2.2.15 --no-document \
+  && gem install bundler -v 2.2.17 --no-document \
```

Comment:
Slightly worried this might pull in a release with a regression that breaks unexpectedly when the docker build cache is busted, making it hard to debug 🤔

Reply:
Yeah I'd prefer to make this an explicit version. Small changes that are not considered breaking for semver might still trip us up (things like error messages etc)

Reply:
Sounds good to me.